frontend/packages/process-editor/src/components/ConfigPanel/ConfigEndEvent/CustomReceiptContent/CreateCustomReceiptForm/SelectCustomReceiptDataModelId/SelectCustomReceiptDataModelId.tsx

```diff
@@ -17,37 +15,33 @@ export const SelectCustomReceiptDataModelId = ({
   onChange,
 }: SelectCustomReceiptDataModelIdProps): React.ReactElement => {
   const { t } = useTranslation();
-  const { layoutSets, existingCustomReceiptLayoutSetId, availableDataModelIds } =
-    useBpmnApiContext();
+  const { layoutSets, existingCustomReceiptLayoutSetId, allDataModelIds } = useBpmnApiContext();
 
   const existingDataModelId: string = getDataTypeFromLayoutSetsWithExistingId(
     layoutSets,
     existingCustomReceiptLayoutSetId,
   );
-
-  const dataModelOptions = getDataModelOptions(availableDataModelIds, existingDataModelId);
-
-  const availableDataModelIdsEmpty: boolean = dataModelOptions.length === 0;
+  const allDataModelIdsEmpty: boolean = allDataModelIds.length === 0;
 
   return (
     <StudioNativeSelect
       label={t('process_editor.configuration_panel_custom_receipt_select_data_model_label')}
       size='small'
       description={
-        availableDataModelIdsEmpty &&
+        allDataModelIdsEmpty &&
         t('process_editor.configuration_panel_custom_receipt_select_data_model_description')
```

Comment:
Had a brief chat with @Annikenkbrathen about using a dynamic description and she generally did not recommend it. And that it was better to list the information as a disabled option in the dropdown 🤔 But I know she was very much involved in this implementation so she might have a different though in this exact case

Reply:
I'll make the component similar to SelectDataType 😄